test/test_analysis_ee_Z_bb.py

```diff
@@ -8,7 +8,8 @@
 
 if context.name == 'fcc':
 
-    from analysis_ee_Z_bb_cfg import config
+    #from analysis_ee_Z_bb_cfg import config
+    from analysis_ee_Z_bb_FCCSW_cfg import config
```

Comment:
this unittest module should not change, but you may add another unittest module to test the reading of papas from FCCSW

Reply:
Have done